src/webparts/footerWebpart/loc/en-us.js

```diff
@@ -5,7 +5,7 @@ define([], function() {
     "DescriptionFieldLabel": "Description Field",
     "AboutUsTitle": "THIS IS US",
     "SiteLinksTitle": "MORE SITE SEEING",
-    "SupportLinksTitle": "NEED A HAND?",
```

Comment:
Where is the need for this update?

Reply:
I don't remember that I delete it. I will check.